react/src/components/atoms/buttons/ButtonWithIcon/ButtonWithIcon.stories.js

```diff
@@ -42,16 +42,11 @@ ButtonWithIconExample.args = {
   capitalized: true,
   'arial-label': '',
   'aria-haspopup': false,
-  name: 'IconChevron',
+  icon: 'IconSearch',
```

Comment:
```suggestion
  icon: 'IconChevron',
```

Reply:
please also change `expanded` to `false` for the default story to match what's in production

Reply:
I think we should also remove the default props in index.js

Reply:
https://mayflower.digital.mass.gov/react/index.html?path=/story/atoms-buttons--buttonwithicon

Reply:
the default prop for `text` needs to exist http://localhost:6006/?path=/docs/atoms-buttons-buttonwithicon--button-search
![Screen Shot 2020-09-24 at 11 37 47 AM](https://user-images.githubusercontent.com/5789411/94167356-6a2bf980-fe5a-11ea-9d0e-9d15a4922d2b.png)

Reply:
@smurrayatwork ^